### PR TITLE
media-fonts/joypixels: Version bump to 6.6.0

### PR DESCRIPTION
**Pull request by mariakeating**

### CODE REVIEW

**Comment by mariakeating** [comment]
what is the way to remove the "no signoff" label? i amended my last commit to fix that mistake but it's still present.

**Comment by thesamesam** [comment]
Can you try putting [please reassign] at the end of the PR title?

File: media-fonts/joypixels/joypixels-6.6.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 Gentoo Authors
```
Comment (juippis) [inline media-fonts/joypixels/joypixels-6.6.0.ebuild:1]: Remember to bump this. We recommend using an automated tool to do bumping, e.g. repoman or pkgdev.

**Review by juippis — APPROVED** [review]
Thanks!

**Pull request closed without merging**